zoom_monitor.py

```diff
@@ -0,0 +1,74 @@
+from contextlib import contextmanager
+import time
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver import Chrome
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+@contextmanager
+def monitor_zoom(url):
+    zoom = ZoomMonitor(url)
+    try:
+        yield zoom
+    finally:
+        zoom.clean_up()
+
+
+class ZoomMonitor():
+    def __init__(self, url):
+        self.driver = Chrome()
+
+        # The links that we receive prompt you to open the Zoom app if it's
+        # available. Replace the domain name to skip that.
+        updated_url = f"https://app.zoom.us/wc/join/{url.split('/')[-1]}"
```

Comment:
This works fine if you've got a Zoom URL, but Google Calendar sometimes gives you a Google link that redirects to Zoom. Maybe we look for that, too, unencode the Zoom URL, and go from there? but I don't have a sense of how difficult that would be. It's something like `https://www.google.com/url?q=https://us05web.zoom.us/j/12344567890?pwd%3Dpasswordgoeshere.1&sa=D&source=calendar&ust=987654321&usg=somethingelsehere`. and I think we want the `q` CGI value where the `%3D` is turned into a `=`. I'll bet there's some kind of url-parsing module we can import, though. Next PR!